attributes/default.rb

```diff
@@ -16,13 +16,14 @@
 #
 
 default['omnibus'].tap do |omnibus|
-  omnibus['build_user']         = 'omnibus'
-  omnibus['build_user_home']    = nil
-  omnibus['install_dir']        = nil
-  omnibus['toolchain_name']     = 'omnibus-toolchain'
-  omnibus['toolchain_version']  = '0.0.1'
-  omnibus['git_version']        = '2.6.2'
-  omnibus['ruby_version']       = '2.1.5'
+  omnibus['build_user']               = 'omnibus'
+  omnibus['build_user_home']          = nil
+  omnibus['install_dir']              = nil
+  omnibus['use_omnibus_toolchain']    = true
```

Comment:
The only reason I added it was for us, to be honest. I like having the safety of being able to 'roll forward' by changing one variable if we end up breaking the world. That being said, I also agree this should not be a choice. I'm fine with removing it if that's our decision.